source/module_base/math_chebyshev.h

```diff
@@ -187,7 +187,7 @@ class Chebyshev
 
 public:
 	//Members:
-    int norder;   // order of Chebyshev expansion
+    int norder{};   // order of Chebyshev expansion
```

Comment:
why not initialize it will 0?

Reply:
It will, please see https://en.cppreference.com/w/cpp/language/zero_initialization

Reply:
I mean, why not write the code "int norder = 0" ?

Reply:
> I mean, why not write the code "int norder = 0" ?

Using bracelet is the only supported grammar for [this fix](https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-default-member-init.html) initializing members in declaration. There is [another fix](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/init-variables.html) available in your expected grammar, but the modifications will be much more.